res/sections/13-Les08032017.tex

```diff
@@ -222,7 +222,8 @@ \subsection{Network Disaster recovery}
 \textit{microwave communication}\footnote{Comunicazione a Microonde.}, faccio
 comunicazione punto punto. Questa tecnologia viene usata abbastanza spesso in
 quanto non richiede una particolare installazione ed è abbastanza economica;
-    \item long-haul network diversity;
+    \item long-haul network diversity: ricorrere a network provider
+    ridondanti;
```

Comment:
Sei sicuro che voglia dire questo? Perchè mi sembra che `ricorrere a network provider ridondanti` sia il `l'alternative routing` in generale